### ISSUE TITLE
nature_recovery model needs retraining: raw scores max at 6.5, no discrimination

### ISSUE THREAD

**Issue by ducroq** [ducroq/llm-distillery#41]
## Problem

The nature_recovery v1 model produces raw scores that max out at 6.52 across 34,977 production articles. 98.6% score below 1.0. The model doesn't discriminate between genuine recovery content and unrelated articles — a hippo culling article scores 3.8 while actual ecosystem restoration barely exceeds 4.0.

## Production score distribution (sadalsuud, 2026-04-15)

```
Total: 34,977 articles
Min: 0.00, Max: 6.52, Mean: 0.09, Std: 0.29
  p25: 0.000, p50: 0.000, p75: 0.055
  p90: 0.269, p95: 0.503, p99: 1.205

[0-1):  34,501 (98.6%)
[1-2):     291 (0.8%)
[2-3):     100 (0.3%)
[3-4):      68 (0.2%)
[4-5):      12 (0.0%)
[5-7):       5 (0.0%)
```

Compare with uplifting v7: scores span 0-10 with a healthy distribution and 73,986 articles above 4.0.

## Impact

- NexusMind #167: percentile normalization inflated these sparse scores (3.8 raw → 9.17 normalized), making garbage articles appear as top recovery content on ovr.news
- Stopgap deployed: normalization disabled for filters with < 200 articles (NexusMind `893cb9a`), so nature_recovery now passes raw scores — lens is effectively empty on ovr.news until model improves

## What needs investigation

1. **Training data quality**: Are the oracle labels (Gemini Flash) scoring nature_recovery correctly? The gatekeeper dimension `recovery_evidence` should prevent non-recovery content from scoring high, but articles like hippo culling (evidence of animal removal, not recovery) slip through
2. **Training set size and diversity**: How many training examples does nature_recovery have vs. uplifting? Is there enough positive-class data?
3. **Score range compression**: The model seems to compress everything into 0-2 range. Is this a calibration issue, a model capacity issue, or a training data issue?
4. **Dimension weights**: Current weights sum to 1.0 with `recovery_evidence` at 0.25 and `measurable_outcomes` at 0.20. Are these appropriate for the content the model actually sees?

## Related

- NexusMind #167 (percentile normalization inflation)
- NexusMind #166 (hippo culling scored 9.2)
- NexusMind #165 (missed obvious recovery article)
- NexusMind #161 (off-topic content in recovery lens)

**Comment by ducroq** [comment]
Fixed in nature_recovery v2 (commit 399d739). Sample weighting (scale=2) + active learning enrichment. Recall@20: 0.55→0.70, NDCG@10: 0.71→0.86. Deployed to Hub, gpu-server, sadalsuud. Normalization pending (NexusMind#169).
